crypto/ec/ec_asn1.c

```diff
@@ -792,18 +794,79 @@ EC_GROUP *EC_GROUP_new_from_ecparameters(const ECPARAMETERS *params)
         goto err;
     }
 
+    /*
+     * Check if the explicit parameters group just created matches one of the
+     * built-in curves.
+     *
+     * We create a copy of the group just built, so that we can remove optional
+     * fields for the lookup: we do this to avoid that one of the optional
+     * parameters is used to force the library into using a less performant and
+     * less secure EC_METHOD instead of the specialized one.
+     * In any case, `seed` is not really used in any computation, while a
+     * cofactor different from the one in the built-in table is just
+     * mathematically wrong anyway and should not be used.
+     */
+    if (NULL == (ctx = BN_CTX_new())) {
```

Comment:
These would normally be written `xxx == NULL`.
I'm okay with the reverse ordering though.

Reply:
My bad, I am a clumsy typist, so I usually style them the other way around so that the compiler can help me detect when I miss the second `=` in the equality operator!

fixup commit coming!

Reply:
Fixed by d3da6adacc333d1ac4db955a02bca29748dd4640